### PR TITLE
Descriptors optimize

### PR DESCRIPTION
**Pull request by z-saj**

### CODE REVIEW

File: aesthetics/fisher/fisher.py [hunk 1]
```diff
@@ -189,4 +188,4 @@ def normalize(fisher_vector):
         is used in step 3, algorithm 1, page 6 of reference [1] """
         v = np.sign(fisher_vector) * np.sqrt(abs(fisher_vector))  # Power normalization
         return v / np.sqrt(np.dot(v, v))  # L2 Normalization
-
+
```
Comment (ylogx) [inline aesthetics/fisher/fisher.py]: Please revert the changes on this line.

File: aesthetics/fisher/fisher.py [hunk 2]
```diff
@@ -7,18 +7,15 @@
 from collections import OrderedDict
 from concurrent.futures import ProcessPoolExecutor
 from scipy.stats import multivariate_normal
-
+descriptors = None
```
Comment (ylogx) [inline aesthetics/fisher/fisher.py]: Lint: There should be one or more lines b/w imports and code

Comment (z-saj) [reply aesthetics/fisher/fisher.py]: Alright 👍

File: aesthetics/fisher/fisher.py (hunk 1 above)
Comment (z-saj) [reply aesthetics/fisher/fisher.py]: Done 👍

File: aesthetics/fisher/fisher.py [hunk 3]
```diff
@@ -8,17 +8,15 @@
 from concurrent.futures import ProcessPoolExecutor
 from scipy.stats import multivariate_normal
 
+descriptors = None
 
 class FisherVector(object):
     """
     Fisher Vector implementation using cv2 v3.2.0+ and python3.
-
```
Comment (ylogx) [inline aesthetics/fisher/fisher.py]: Need these new lines for proper doc formatting

File: aesthetics/fisher/fisher.py [hunk 4]
```diff
@@ -189,4 +189,3 @@ def normalize(fisher_vector):
         is used in step 3, algorithm 1, page 6 of reference [1] """
         v = np.sign(fisher_vector) * np.sqrt(abs(fisher_vector))  # Power normalization
         return v / np.sqrt(np.dot(v, v))  # L2 Normalization
-
```
Comment (ylogx) [inline aesthetics/fisher/fisher.py]: +1

**Comment by z-saj** [comment]
Added broadcasting and list comprehension in place of for loop. 
~ 5% improvement in time.

File: aesthetics/fisher/fisher.py (hunk 1 above)
Comment (z-saj) [reply aesthetics/fisher/fisher.py]: Makes code ~10% faster
